src/pages/schedule-page.js

```diff
@@ -100,7 +100,7 @@ const data = [
     { id: 16, module: 'Professional Practice', moduleCode: 'CS1456', hall: 'B202', studentCount: '40 users', time: '10:30 AM' },
 ];
 
-const rowsPerPage = 5;
+const rowsPerPage = 7;
```

Comment:
Any specific reason to change this since this is not related to error page?

Reply:
yes, 7 will be the right size to fill the page